modules/OCPP/OCPP.hpp

```diff
@@ -57,7 +59,19 @@ class OCPP : public Everest::ModuleBase {
     const std::unique_ptr<ocpp_1_6_charge_pointImplBase> p_main;
     const std::unique_ptr<auth_token_validatorImplBase> p_auth_validator;
     const std::vector<std::unique_ptr<evse_managerIntf>> r_evse_manager;
-
+    std::map<int32_t, int32_t> ResConnMap;
```

Comment:
I think we should continue using snake case for consistency here. eg. res_conn_map and res_state_map. Maybe drop the _map postfix as well